src/conmon.c

```diff
@@ -84,6 +84,15 @@ int main(int argc, char *argv[])
 		exit(0);
 	}
 
+	if (getenv("NOTIFY_SOCKET") != NULL) {
```

Comment:
idk what's involved in a getenv call, but maybe we should save the value here and use it below?